cmd/legacy_main_test.go

```diff
@@ -52,7 +51,9 @@ func (t *MainTest) TestCreateStorageHandle() {
 		AppName:             "app",
```

Comment:
AppName should also move to new config. Same for TestCreateStorageHandle_WithClientProtocolAsGRPC Test

Reply:
AppName wasn't required. So removed it.